### ISSUE TITLE
Should compass be allowed to run with no validators?

### ISSUE THREAD

**Issue by GalRogozinski** [iotaledger/compass#76]
### Discussion
Currently compass can run with no external validators, and rely only on the validation of its direct IRI peer. We currently are waiting for cIRI to also have an API that will do extra validation.

Once that is ready, maybe we want to force compass to have an external validator???
On one hand people running private tangles may not like that, on the other hand it is safer this way...

**Comment by alon-e** [comment]
I think it should - running with validators by default makes the setting up a private tangle more complicated.
However, I think we should have more information on why `validators` are a good thing and who to set them up - opening a new issue on this.
